Feature: Update Icons (selfhst repo)

bannert1337 opened the pull request:
## ✍️ Description  
I am in the process of updating all icons to SVG if available, else WEBP, to improve quality, performance and efficiency of the website. 

## 🔗 Related PR / Discussion / Issue  

Link: #

## ✅ Prerequisites  

Before this PR can be reviewed, the following must be completed:  

- [X] **Self-review performed** – Code follows established patterns and conventions.  
- [X] **Testing performed** – Changes have been thoroughly tested and verified.  

## 🛠️ Type of Change  

Select all that apply:

- [ ] 🆕 **New script** – A fully functional and tested script or script set.
- [ ] 🐞 **Bug fix**  – Resolves an issue without breaking functionality.  
- [ ] ✨ **New feature**  – Adds new, non-breaking functionality.  
- [ ] 💥 **Breaking change**  – Alters existing functionality in a way that may require updates.  

## 📋 Additional Information (optional)  
<!-- Provide extra context, screenshots, or references if needed. -->

bannert1337 committed:
chore(icons): :zap: Replace Actual icon png with svg

Replaced the PNG icon of Actual with a SVG from selfhst/icons

bannert1337 committed:
chore(icons): :zap: Replace Proxmox VE icon PNG with SVG

Repalce all PNG icons for Proxmox VE with SVG from selfhst/icons

bannert1337 committed:
chore(icons): :zap: Replace AdGuard PNG icon with SVG

Replace the PNG icon of AdGuard Home with SVG from selfhst/icons

bannert1337 committed:
chore(icons): update multiple icons to SVG

Replaced rasterized images with vectorized SVG for better efficiency, sharpness, performance and lower size

bannert1337 committed:
Merge remote-tracking branch 'origin/icon-updates' into icon-updates

# Conflicts:
#	json/opnsense-vm.json

bannert1337 committed:
chore(icons): update logo URLs for multiple services to use selfhst/icons repository

Updated logo URLs in JSON configuration files for the following services:
- WatchYourLAN
- Authelia
- Zoraxy
- Authentik
- Pi-hole
- Radicale
- Pocket ID
- Keycloak
- Grafana
- MongoDB
- MySQL
- phpIPAM
- Duplicati
- UrBackup
- Technitium DNS
- Seelf
- Checkmk
- Glance
- ZeroTier
- BunkerWeb

MickLesk commented:
Do you have the time and inclination to do this for all the scripts we have? Say if any are missing -> Request to the repo. Then we can treat them all the same. Feel free to link the maintainer here :D

bannert1337 commented:
Yeah, I will do all of them. Can also maintain them if you like!

bannert1337 committed:
chore(icons): update logo URLs for multiple services to use selfhst/icons repository

Updated logo URLs in JSON configuration files for the following services:
- Uptime Kuma
- Umami
- Apache Tomcat

bannert1337 committed:
chore(icons): update application logos to use selfhst/icons repository

bannert1337 committed:
chore(icons): update application logos to use selfhst/icons repository

MickLesk commented:
Mention us (@community-scripts/contributor ) If you are ready and we can Check/Approve

bannert1337 commented:
@community-scripts/contributor
For the beginning I have replaced all icons available in the selfhst icons repository. I will look up the remaining icons tomorrow and create another PR

bannert1337 commented:
Looks really clean and already a lot faster. There are some SVG that load some milliseconds and the still rasterized, non WebP are still loading much slower. Would say major step towards quality.

bannert1337 commented:
Some icons would really benefit from dedicated light and dark mode icons which are selected automatically based on the active theme. But I have not investigated, how that would need to be implemented in the current system.

shollyethan commented:
> Just noticed a mistake. Prowlarr has the Radarr icon, copy-paste mistake probably. I will make a quick commit ready.

Hey! I'm the maintainer of selfh.st/icons and would also love to contribute where possible.

I'm currently in the process of creating light and dark versions for as many icons as possible. Many can already be found in the repo itself, or you can programatically access them here:

https://github.com/selfhst/cdn/blob/main/directory/icons.json

I'll be adding a key for dark icons in the next week or two, but in the meantime, the JSON file should be a helpful reference for available formats and light versions.

Feel free to also ping me with any I'm missing and I'll work to get them added ASAP.

CrazyWolf13 commented:
@shollyethan  and @bannert1337 

Thanks you two! Really appreciate that!

bannert1337 commented:
@shollyethan i made a PR to add a Apache Tika icon:
https://github.com/selfhst/icons/pull/274

MickLesk commented:
@shollyethan thx for your work!

ive made an "list" of our scripts, Should I split them up and create them individually in “Discussions”? 

looks like so:
![image](https://github.com/user-attachments/assets/55ae430a-6891-45d5-87fb-393c33222880)

bannert1337 commented:
@MickLesk Is this list supposed to contain all scripts without an SVG icon currently?

MickLesk commented:
> @MickLesk Is this list supposed to contain all scripts without an SVG icon currently?

i crawl the list with ALL Logos that not use the selfhst repo :D
